### PR TITLE
Custom mode pipe delimiter: section count mode

### LINKED ISSUE #3742: Custom mode pipe delimiter: section count mode

### PR DESCRIPTION
**Pull request by albertying**
### Description

A new feature in custom mode that allows users to create tests with a variable number of sections separated by the pipe delimiter.

Closes #3742

### CODE REVIEW

File: frontend/static/html/popups.html [hunk 1]
```diff
@@ -477,6 +477,10 @@
           Time
           <input type="number" value="" min="1" max="10000" />
         </label>
+        <label class="sectioncount hidden" id="wowzers">
```
Comment (Miodec) [inline frontend/static/html/popups.html:480]: id wowzers?

Comment (albertying) [reply frontend/static/html/popups.html]: Sorry about that, I was testing something and forgot to remove it. I like to use random names so I don't have to think about them too much.

**Comment by albertying** [comment]
I just realized I broke the plus one/two/three funbox with the commit ca7cab0.

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/4522829358

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/4523034358

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/4548063752

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/4548219753

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/4548237932

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/4559044701

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/4947488296

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/4947488296

**Pull request merged**